exp-player/addon/components/exp-card-sort/template.hbs

```diff
@@ -19,10 +19,17 @@
   </div>
   <h4 class="center">{{cards.length}} {{t 'qsort.sections.1.itemsLeft'}}</h4>
 {{/if}}
+<div class="row hidden-xs">
+  {{#each buckets as |bucket|}}
+    <div class="col-sm-4">
+      <h3 class="bucket-label" style={{height}}>{{t bucket.name}}</h3>
+    </div>
+  {{/each}}
+</div>
 <div class="row">
     {{#each buckets as |bucket|}}
       <div class="col-sm-4">
-          <h3 class="text-center">{{t bucket.name}}</h3>
+          <h3 class="bucket-label visible-xs" style={{height}}>{{t bucket.name}}</h3>
```

Comment:
For future note: Why is this a heading? Headings imply a structure to the document, but this appears to be just a label.

We should generally avoid using heading tags for pure formatting; it confuses screen readers and makes it harder for vision-impaired users to use the site.